app/Modules/Gateway/Gateways/PaypalStandard.php

```diff
@@ -72,6 +74,19 @@ private function initActions()
         add_action('smartpay_paypal_web_accept', [$this, 'process_smartpay_paypal_web_accept'], 10, 2);
     }
 
+	public function addWarningMessage( Payment $payment ): void {
+		if ( isset( $_GET['PayerID'] ) && $payment->status === 'Pending' && $payment->gateway === 'paypal' ) {
```

Comment:
How many payment gateways do we have? like checked here `$payment->gateway === 'paypal'`.

Reply:
I mean can we move those gateways identifiers to constants? if there is more than `'paypal'`?

Reply:
We have several payment gateways. We add this method to the action which is declared on the payment receipt page. So, every payment receipt it will be triggered but the issue is with PayPal. That is why I have added the condition.

Reply:
I meant the gateway and status keys 😆 Can we use constants here?

ref: https://github.com/wpsmartpay/wp-smartpay/blob/9beac8da11132cb6a1aeed26fbad3538960cc647/resources/js/utils/constant.js#L6

Reply:
Yeah, we can. I am going to use the constants of statuses at this moment. But for the payment gateway name, I will use it in the later version because need some refactoring.

Reply:
![image](https://github.com/wpsmartpay/wp-smartpay/assets/57439878/d678ba4f-c131-41ed-a6fe-347757812b63)

Reply:
Okay, LGTM